docs/collectors/ElasticSearchCollector.md

```diff
@@ -28,11 +28,10 @@ measure_collector_time | False | Collect the collector run time in ms | bool
 metrics_blacklist | None | Regex to match metrics to block. Mutually exclusive with metrics_whitelist | NoneType
 metrics_whitelist | None | Regex to match metrics to transmit. Mutually exclusive with metrics_blacklist | NoneType
 port | 9200 |  | int
-stats | jvm, thread_pool, indices, | Available stats:<br>
- - jvm (JVM information)<br>
- - thread_pool (Thread pool information)<br>
- - indices (Individual index stats)<br>
- | list
+stats | jvm | JVM information | int |
+stats | thread_pool | Thread pool information|int |
+stats | indices| Individual index stats| int | 
+stats | cluster | Cluster stats: nodes total, nodes data,shards active primary,shards active,shards relocating,shards unassigned,shards dealyed unassigned,fetch inflight and pending tasks | int |
```

Comment:
Can you auto generate the doc instead of updating manually?
info should be added to the collector in `get_default_config_help`